src/components/reviews/reviewsList.jsx

```diff
@@ -47,34 +47,41 @@ class ReviewsList extends React.Component {
   }
 
   render() {
-    var tiles = [];
+    var reviews = [];
```

Comment:
The use of reviews as a variable instead of tiles makes things a little clearer on what you are actually rendering here.

Reply:
agreed!